botfront/cypress/integration/nlu_models/nlu_models.spec.js

```diff
@@ -1,143 +1,75 @@
-        
-            cy.get('[data-cy=offline-model]').click();
-            cy.get('[data-cy=confirm-popup]').contains('Yes').click();
-            checkPublishingState(1, 1);
-            cy.get('[data-cy=nlu-model-card]').first().get('[data-cy=online-model]');
-
-            deleteSecondModel();
-            checkPublishingState(1, 0);
-        });
+        // Try to delete the default model
+        cy.visit(`/project/${this.bf_project_id}/nlu/models`);
+        cy.get('.nlu-menu-settings').click();
+        cy.contains('Delete').click();
+        cy.get('.dowload-model-backup-button').click();
+        cy.get('.delete-model-button').should('be.disabled');
     });
 
-    describe('NLU Import', function() {
-        it('Should import a json file with training data', function() {
-            this.dropEvent = {
-                dataTransfer: {
-                    files: [{ path: `${Cypress.config('fixturesFolder')}/nlu_import.json` }],
-                },
-            };
-            this.dropEvent.force = true; // https://github.com/cypress-io/cypress/issues/914
-            cy.visit(`/project/${this.bf_project_id}/nlu/models`);
-            cy.contains('French').click();
-            cy.get(':nth-child(1) > .extra > .basic > .primary').click();
-            cy.get('.nlu-menu-settings').click();
-            cy.contains('Import').click();
-            cy.fixture('nlu_import.json', 'utf8').then((content) => {
-                cy.get('.file-dropzone').upload(content, 'data.json');
-            });
-
-            cy.contains('Import Training Data').click();
+    it('should NOT be able to call the nlu.remove for the default model', function() {
+        cy.MeteorCall('nlu.remove', [
+            this.bf_model_id,
+            this.bf_project_id,
+        ]).then((result) => {
+            expect(result).equals('Model Not Deleted');
         });
     });
 
-    describe('Model deletion', function() {
-        it('Should delete the model parameters', function() {
-            cy.visit(`/project/${this.bf_project_id}/nlu/models`);
-            cy.contains('French').click();
-            // TODO add instance when tests do instances
-            // Save
-            cy.get(`#model-${modelName} [data-cy=open-model]`)
-                .first()
-                .click();
-            cy.get('.nlu-menu-settings').click();
-            cy.contains('Delete').click();
-            cy.get('.nlu-menu-settings').click();
-            cy.get('.delete-model-button').should('be.disabled');
-            cy.get('.dowload-model-backup-button').click();
-            cy.get('.delete-model-button').should('be.enabled');
-            cy.get('.delete-model-button').click();
-            cy.get('.ui.page.modals').should('be.visible');
-            cy.get('.ui.page.modals .primary').click();
-            cy.url('should.be', `/project/${this.bf_project_id}/nlu/models`);
-            cy.get('.s-alert-success').should('be.visible');
```

Comment:
Why is model deletion not tested anymore?

Reply:
Will add this back

Reply:
It is already there in this test 'should be able to create a new Model by adding a new language, to the project and then delete it'